Sources/Shared/Data Manager/Model/Documents/File.m

```diff
@@ -74,8 +78,54 @@ - (void)dealloc {
     [_creator release];
     [_dateCreated release];
     [_dateModified release];
+    _naturalName = nil;
     [super dealloc];
 }
 
+-(NSString *) getNaturalName{
```

Comment:
Add a comment (with Xcode syntax) that explains when this method returns the natural name, and when it simply returns the name. Link it with the method convertToNaturalName.